classes/Mail.php

```diff
@@ -381,10 +381,21 @@ function ($carry, $item) {
             ) {
                 $moduleName = $res[1];
             }
+            
+            //It template path is passed it is not overridden with standard template path 
```

Comment:
Sorry you will hate me --'
```php
 // If $templatePath parameter is used, it is not overridden with standard template path 
```

Reply:
Otherwise everything looks ok for me @Quetzacoalt91 @mickaelandrieu wdyt?

Reply:
No hate at all! :) you're right, this comment is more appropriate.